src/platform/packages/private/kbn-grid-layout/grid/use_grid_layout_events/types.ts

```diff
@@ -8,3 +8,8 @@
  */
 
 export type UserInteractionEvent = React.UIEvent<HTMLElement> | Event;
+
+export interface MousePosition {
```

Comment:
```suggestion
export interface PointerPosition {
```
Nit: Since this can be used for both mouse and touch events, it might be better to have a more inclusive name. That said, MousePosition reads more naturally in code, making it easier to understand. No strong preference, just something to consider!

Reply:
Good call! Done in d47dbf0a42305f3dcad745e6f9cf21097db0669e